c10/util/order_preserving_flat_hash_map.h

```diff
@@ -736,11 +750,10 @@ class sherwood_v3_table : private EntryAlloc, private Hasher, private Equal
 
     void clear()
     {
-        for (EntryPointer it = entries, end = it + static_cast<ptrdiff_t>(num_slots_minus_one + max_lookups); it != end; ++it)
-        {
-            if (it->has_value())
-                it->destroy_value();
+        for (EntryPointer it = sentinel->next; it != sentinel; it = it->next) {
```

Comment:
Iteration via linked list is going to be much slower, probably want to leave this as is.